extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -224,30 +225,24 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
     }
   };
 
-  private addNamesToMsg = async (msg: SendableMsg): Promise<void> => {
+  private addNameToEmail = async (email: string): Promise<string> => {
```

Comment:
Let's call this `formatSenderEmailAsMimeString` (is that what it does?)

Reply:
done